docs/nidigital/class.rst

```diff
@@ -1668,7 +1668,11 @@ initiate
 
     .. py:method:: initiate()
 
-            TBD
+            Starts bursting the pattern, causing the NI-Digital session
+            to be committed. To stop the pattern burst, call abort
+            method. If keep alive pattern is bursting when abort
```

Comment:
@sbethur I feel like we don't need to call out the context manager here, because we have a note that calls it out already: "This method will return a Python context manager that will initiate on entering and abort on exit." Just calling the that abort will not stop an in-progress keep alive pattern feels sufficient.

Reply:
agreed